sea_itt_lib/Utils.cpp

```diff
@@ -26,8 +26,8 @@
     #include <cxxabi.h>
     #include <dlfcn.h>
 
-    #ifndef __ANDROID__
-        //#include <execinfo.h>
+    #if !defined(__ANDROID__) && !defined(USE_MUSL_LIBC)
+        #include <execinfo.h>
```

Comment:
Why was this commented out before and now it is not. If USE_MUSL_LIBC is not defined should this code really be included?